src/mds/MDLog.h

```diff
@@ -301,5 +301,10 @@ class MDLog {
   std::set<LogSegment*> expired_segments;
   std::set<LogSegment*> expiring_segments;
   uint64_t events_since_last_major_segment = 0;
+  // -- helper functions
+  void _log_estimated_complete_time(utime_t start_time, uint64_t start_pos);
+
+  // -- events --
+  LogEvent *cur_event = nullptr;
```

Comment:
Where is `cur_event` used? I think this was done away with Patrick's changes for minor log segments (PR #48732), so, this looks like this happened due to an incorrect rebase, yes?

Reply:
Yes